generator/transformers/common_producer.py

```diff
@@ -309,6 +309,8 @@ def extract_type(self, param: Param) -> dict:
 
         if not param.is_mandatory and re.match(r'\w*Int\d*|BOOL', data['type_native']):
             data['type_native'] = data['type_sdl']
+        elif not param.is_mandatory and re.match(r'\w*float\d*', data['type_native']):
```

Comment:
Can this `elif` be combined into one statement with the above `if` like so: `re.match(r'\w*Int\d*|BOOL|\w*float\d*', data['type_native'])`?